apps/expo-go/android/expoview/src/main/java/host/exp/exponent/storage/ExponentInstallationId.kt

```diff
@@ -38,6 +38,8 @@ internal class ExponentInstallationId(
       // do nothing, try other sources
     } catch (e: IllegalArgumentException) {
       // do nothing, try other sources
+    } catch (e: NullPointerException) {
+      // do nothing, try other sources
     }
```

Comment:
We could collapse all catches into one using catching `Throwable` instead. What do you think?